src/app/home/home.component.ts

```diff
@@ -78,9 +77,9 @@ export class HomeComponent implements AfterViewInit, OnDestroy {
             next: () => console.log('pinging server...'),
             error: (e: HttpErrorResponse) => {
                 if (e.status === 429) {
-                    this.disableEditorToManyRequests();
+                    this.disableEditor('Tepër kërkesa për shenjime për momentin')
                 } else {
-                    this.disableEditor();
+                    this.disableEditor('Fatkeqësisht kemi një problem me serverat. Ju kërkojmë ndjesë, ndërsa kërkojme për një zgjidhje.' );
                 }
```

Comment:
How about we move this if/else here (which might grow on new codes) to the `disableEditor` method, and pass as an argument the status code, whichever that might be?